package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-transitive-number",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I usually bump versions in a separate commit (outside of PR) using `npm version`. Also, React has their minor version bumped, so I believe we should do that too.